### PR TITLE
Drop support for legacy Python 2.7

### LINKED ISSUE #120: Dropping python 2
What's the timeline for dropping support for python 2? Python 2 formally dies in 2020 and Astropy dropped it long ago. Removing the `six` and `__future__` code will make things a lot cleaner.

Should we issue a v1.0 on PyPI and drop python 2 after that?

### PR DESCRIPTION
**Pull request by hugovk**
Fixes #120.

### CODE REVIEW

File: ah_bootstrap.py [hunk 1]
```diff
@@ -157,7 +157,7 @@
 ]
 
 
-class _Bootstrapper(object):
```
Comment (bsipocz) [inline ah_bootstrap.py]: Please do not change external files, like `ah_bootstrap.py`. The file will become python3 once we switch to astropy_helpers v3.2.x

Comment (hugovk) [reply ah_bootstrap.py]: Thanks, I've reverted `ah_bootstrap.py`.

**Comment by cbanek** [comment]
First off, thank you so much for your contribution.  I really appreciate it!  I'm not an expert in python2 to python3 migrations, but I'll review it.  It also looks like we should probably add an entry to the CHANGES.rst (that's what the bot is complaining about).  Feel free to add that in.

File: pyvo/dal/dbapi2.py [hunk 2]
```diff
@@ -18,14 +12,14 @@
 __all__ = "STRING BINARY NUMBER DATETIME ROWID".split()
 
 
-class Error(StandardError):
+class Error(Exception):
```
Comment (bsipocz) [inline pyvo/dal/dbapi2.py]: beyond the scope of this PR, but I'm not sure what use of these base classes have, why not use the standard python `Warning` and a sensible Exception class to subclass from?

**Comment by bsipocz** [comment]
I suspect there may be some other stuff that can be cleaned up (there is always), but all changes here looks good.

**Comment by cbanek** [comment]
Okay everything seems good once we get a changes.rst entry!

**Comment by hugovk** [comment]
CHANGES.rst updated!

**Comment by cbanek** [comment]
Awesome, thanks so much for your help!

**Pull request merged**